lib/logflare/source/recent_logs_server.ex

```diff
@@ -191,10 +92,6 @@ defmodule Logflare.Source.RecentLogsServer do
   end
 
   ## Private Functions
-  defp do_push({:push, _source_token, [le | _] = log_events}, state) do
-    new_queue = (Enum.to_list(state.recent) ++ log_events) |> LQueue.from_list(100)
```

Comment:
Yeah I think this was a huge GC offender. Take a look at the LQueue source code.

Reply:
omg its in an Agent 🤯

Reply:
Yeah fml